chart/values.yaml

```diff
@@ -706,6 +706,7 @@ config:
     rbac: 'True'
   celery:
     default_queue: celery
+    worker_concurrency: 8
```

Comment:
The default is 16 and this changes it to 8

Reply:
I know about this as I mentionned in the PR description:

> The only impact is that with this PR the default concurrency of KEDA goes down from 16 to 8 (I thought that would be better than to set the concurrency by default from 8 to 16 for all CeleryExecutor deployments).

I did this because the default value in airflow settings seems to be 8: https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg#L666

I am definitely ok setting back this to 16 if you want too.

Reply:
Let's keep that 16 please -- https://github.com/apache/airflow/pull/13612 (This PR will change the default setting in default_airflow.cfg) back to 16

Reply:
Done.